solr/core/src/java/org/apache/solr/schema/DenseVectorField.java

```diff
@@ -394,7 +394,7 @@ public Query getFieldQuery(QParser parser, SchemaField field, String externalVal
 
   /** Not Supported */
```

Comment:
Wondering if we might wanna
```
@Override
protected boolean treatUnboundedRangeAsExistence(SchemaField field) {
    return false;
}
```
to retain the behaviour of `getRangeQuery` throwing an exception.

Reply:
Answering own question: it already returns `false` via this implementation in the class hierarchy: https://github.com/apache/solr/blob/releases/solr/9.7.0/solr/core/src/java/org/apache/solr/schema/NumericFieldType.java#L360